three_values.py

```diff
@@ -0,0 +1,33 @@
+from sorting_methods import sortowanie_babelkowe
+from sorting_methods import sortowanie_wstawianie
+import datetime
+
+def index3(data,function):
+    list=data[0]
+    min=data[1]
+    index=[]
+    j=0
+    
+    for i in range(len(list)):
+        if list[i] > min and j<3:
+            index.append(i)
+            j+=1
+
+    index_result=function([list[index[0]], list[index[1]], list[index[2]]])
+    result=[]
+
+    for i in range(3):
+        result.append(list.index(index_result[i]))
+    return result
+
+list_=([9,1,1,1,0,0,0,8,10,11,14,2132323,24234524], 2)
+
+start = datetime.datetime.now()
+print(index3(list_,sortowanie_babelkowe))
```

Comment:
dlaczego "list_" po pierwsze nie jest to żadna lista tylko dane złożone z tablicy i wartości progowej, po drugie podkreślnik za nazwą zmiennej? To jakaś konwencja i tak się robi czy radosna twórczość?
Lepiej byłoby to nazwać po prostu data.